src/Themes/HackathonTheme/Views/Widget-VAlert.liquid

```diff
@@ -0,0 +1,14 @@
+{% assign color = Model.ContentItem.Content.VAlert.Color.Text %}
+{% assign type = Model.ContentItem.Content.VAlert.Type.Text %}
+{% assign elevation = Model.ContentItem.Content.VAlert.Elevation.Value %}
+{% assign icon = Model.ContentItem.Content.VAlert.Icon.Text %}
+{% assign border = Model.ContentItem.Content.VAlert.Border.Text %}
+
+<v-alert {{ Model.ContentItem.Content.VAlert.Props.Values | join: ' ' | raw }}
+  {% if color %}color="{{ color }}"{% endif %}
+  {% if type %}type="{{ type }}"{% endif %}
+  {% if elevation %}elevation="{{ elevation }}"{% endif %}
+  {% if icon %}icon="{{ icon }}"{% endif %}
+  {% if border %}border="{{ border }}"{% endif %}>
+  {{ Model.ContentItem.Content.HtmlBodyPart.Html | sanitize_html | raw }}
```

Comment:
Here, I suggest you simply render the HtmlBodyPart.
`{{ Model.Content.HtmlBodyPart | shape_render }}`